### PR TITLE
Boostr updates

### PR DESCRIPTION
**Pull request by cavis**

### CODE REVIEW

File: src/make_copies.js [hunk 1]
```diff
@@ -65,10 +65,18 @@ module.exports = async function main(
                 /%RANGE_START_ISO/g,
                 config.inclusiveRangeStart.toISOString()
               )
+              .replace(
+                /%RANGE_START_DATE_ISO/g,
```
Comment (farski) [inline src/make_copies.js:69]: These directives need to be added to the readme

File: src/query-jobs/boostr_impressions.js [hunk 2]
```diff
@@ -6,7 +6,7 @@
 module.exports = async function main(config) {
   const query = `
     SELECT
-      timestamp_trunc(timestamp, day) AS Date,
+      FORMAT_TIMESTAMP("%m/%d/%Y", timestamp) as Date,
```
Comment (farski) [inline src/query-jobs/boostr_impressions.js:9]: Use `AS` for consistent formatting

Comment (farski) [reply src/query-jobs/boostr_impressions.js:9]: m/d/y is probably the most ambiguous date format available, so using it in a structured report seems risky

**Review by farski — CHANGES_REQUESTED** [review]

File: src/query-jobs/boostr_impressions.js (hunk 2 above)
Comment (cavis) [reply src/query-jobs/boostr_impressions.js:9]: Right?  I asked that same question (after they asked me to change it), but apparently this is the format they ingest.

File: src/make_copies.js (hunk 1 above)
Comment (cavis) [reply src/make_copies.js:69]: Added!

**Review by farski — APPROVED** [review]

**Pull request merged**